app/helpers/container_summary_helper.rb

```diff
@@ -126,14 +126,21 @@ def textual_collection_link(collection, as: nil, feature: nil)
 
     if count > 0 && role_allows(:feature => feature)
       if collection.respond_to?(:proxy_association)
-        h[:link] = url_for(:action  => 'show',
-                           :id      => collection.proxy_association.owner,
-                           :display => collection.proxy_association.reflection.name)
+        h[:title] = "Show all #{label}"
+        if collection.count == 1
```

Comment:
These conditionals should be nested the other way around; we should decide whether to link to the collection vs a single object, before we try to work out exactly how to link to the collection.

(That'll also fix the fact you've lost the `:title` for the outer else clause)